frontend/src/app/main/ui/workspace/viewport/outline.cljs

```diff
@@ -34,12 +35,15 @@
             (or (ex/ignoring (upf/format-path (:content shape)))
                 "")))
 
-        {:keys [x y width height selrect]} shape
+        {:keys [x y width height selrect rx ry]} shape
+
+        style-attrs (attrs/extract-style-attrs shape)
```

Comment:
`extract-style-attrs` is pretty expensive for just extract the `d` if it is found for the outline component.
We need to make the `d` calculation (with rounding, etc..) code more reusable and use it here instead of calling the `extract-style-attrs` for a single attribute

Reply:
Fixed